assets/js/index.js

```diff
@@ -1,26 +1,173 @@
+    lang: {
+      lang1: 'html',
+      lang2: 'css',
+      lang3: 'js',
+      lang4: 'ruby',
+    },
+    category: 'project1',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    list: {
+      list1: 'FACEBOOK',
+      list2: 'Full Stack Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/four.png',
+    desc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project2',
+  },
+  {
+    id: 3,
+    title: 'Facebook 360',
+    list: {
+      list1: 'FACEBOOK',
+      list2: 'Full Stack Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/Snapshoot Portfolio.png',
+    desc: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project3',
+  },
+  {
+    id: 4,
+    title: 'Uber',
+    list: {
+      list1: 'Uber',
+      list2: 'Lead Developer',
+      list3: '2015',
+    },
+    image: 'static/imgs/two.png',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project4',
+  },
+];
+
+// Add burger==========================================================================
+const burger = document.querySelector(
+  '.header__container__parhead__nav__ul__responsive',
+);
+const iwq = document.querySelector('.iwq');
+const parpopup = document.querySelector('.parpopup');
+const parpopuPopup = document.querySelector('.parpopup__popup');
 const navLinks = document.querySelectorAll('.mobile-nav-links');
 const email = document.querySelector('.form__control > input#email');
 const form = document.querySelector('form');
-const emailError = document.querySelector('#emailerror');
-
+const emailerror = document.querySelector('#emailerror');
+const pardetail = document.querySelector('.pardetail');
+// Add eventlistener===========================================================================
 burger.addEventListener('click', () => {
-  burger.classList.toggle('change');
-  mobileNav.classList.toggle('show');
+  if (burger.classList.contains('change')) {
+    burger.classList.remove('change');
+  } else {
+    burger.classList.add('change');
+  }
+
+  if (iwq.classList.contains('show')) {
+    iwq.classList.remove('show');
+  } else {
+    iwq.classList.add('show');
+  }
 });
+// Add function details============================================================================
+
+function detail(c) {
+  parpopup.style.display = 'flex';
```

Comment:
- [ ] I noticed a few style discrepancies in the popup window that need to be fixed. Firstly, the `title` appears small and lacks the appropriate `font size, font weight, and color`. :100: 
- [ ] Secondly, the text `Canopy, Back end dev, 2015` and the programming languages need to be vertically aligned with the other contents of the popup window and styled properly as seen in the original design. :+1: 
- [ ] Additionally, the image link is currently broken and needs to be fixed. Lastly, the buttons need to be styled properly so that the entire popup window matches the design guidelines. Also, please fix the icon in the `See Live` button. :wink: 
- [ ] Kindly ensure that the background overlay of the popup is the same color as in the Figma design :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/233317055-b650875a-eece-4630-b5fd-e6c7396ef4d9.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/233317556-a9c9a06e-2dbd-41b7-ba6f-b434c75de1c3.png)

- [ ] Even though the design on the desktop follows the wireframe, just as I mentioned before in the mobile view all the changes requested may be done for the desktop view as well.
- [ ] Aside from that, the popup card should look bigger as seen in the Figma design and the background overlay of the popup should be the same color and blurred as in the Figma design as I mentioned before :+1: 
- [ ] Please refer to the design guidelines and ensure that you follow the given design closely:100: 

- Your Design
![image](https://user-images.githubusercontent.com/102699958/233319890-4bec15eb-a951-4d24-bbcc-114566e43e7b.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/233320262-a0639000-674a-4e5e-9b52-689964eae892.png)

Reply:
Resolve